### ISSUE TITLE
Make generation of results button visible from different tabs

### ISSUE THREAD

**Issue by mert0248** [thibautjombart/covid19_bed_occupancy#57]
Not at all critical but maybe would be nice if the "Generate results" button could persist across the "Length of Stay Distribution" "Doubling time distribution" and "Main results" tabs so that you can always see where to press to run the simulation - and if not on the "Main Results" tab then switch focus to there?

Also is there any way to give an a visual indication that a calculation is in progress when it takes more than a second or two?

**Comment by samclifford** [comment]
Calculation in progress is in issue #59 

The simulation results are only relevant on the "Main Results" tab, though. The distributions are visualisations of model inputs not outputs. I'm not sure what additional use having it persistent would be given that pressing it when "Main Results" hasn't got focus will no provide no visual clue that anything has happened.

**Comment by mert0248** [comment]
Yes agree, hence thinking  it might also switch focus? I think main thing was when fiddling with some of the input distribution parameters I found myself thinking “where’s that button to see the results”. Happy to accept that might just be me :-)

**Comment by thibautjombart** [comment]
No I think that's a natural behaviour.

**Comment by samclifford** [comment]
Can we close this now, given the redesign of the tabs in #75?
